eslint-plugin-kuc-v1/validator-in-update.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  create: function(context) {
+    return {
+      ClassDeclaration: function(node) {
+        const superClass = node.superClass.name;
+        if (superClass !== "KucBase") return;
+
+        const nodeBody = node.body;
+        if (nodeBody.type !== "ClassBody") return;
```

Comment:
In addition to these ignored patterns, I think we can ignore them below, too.
- If developer doesn't import from "../base/validator" or "../../base/validator", etc.